src/main/webapp/editFilesFragment.xhtml

```diff
@@ -377,7 +276,6 @@
                     
                     <p:message for="fileUpload" id="uploadMessage" display="text" />
                     
-                    <p:commandButton id="uploadStarted" action="#{EditDatafilesPage.uploadStarted()}" update="" style="display:none"/>
```

Comment:
This was part of a legacy fix/hack... But it looks like we are not using it anymore for anything useful... So should be ok to remove as well.